src/styles/main/_scorecard.scss

```diff
@@ -753,7 +753,7 @@ $sc-font-lgr: 3rem;
           margin-left: 15px;
           line-height: 1.2;
           font-size: 12px;
-          color: #0047BA;
+          color: $blue;
```

Comment:
why is the scorecard involved in this change at all? Also, this should probably be `$theme-color`... let's work at a higher level of meaning so we don't interfere with theming.